packages/block-editor/src/components/inserter-list-item/index.js

```diff
@@ -20,11 +16,12 @@ import {
  * Internal dependencies
  */
 import BlockIcon from '../block-icon';
+import { InserterListboxItem } from '../inserter-listbox';
 import InserterDraggableBlocks from '../inserter-draggable-blocks';
 
 function InserterListItem( {
 	className,
-	composite,
+	isFirst,
```

Comment:
`InserterListItem` isn't exposed as part of the public API so it's fine to change 👍🏻